pkg/interface/src/views/landscape/components/Sidebar/SidebarListHeader.tsx

```diff
@@ -54,7 +54,7 @@ export function SidebarListHeader(props: {
   const feedPath = groupPath ? getFeedPath(associations.groups[groupPath]) : undefined;
 
   const unreadCount = useHarkState(
-    s => s.unreads?.graph?.[feedPath ?? '']?.['/']?.unreads as number ?? 0
+    s => s.unreads?.[`/graph/${feedPath.slice(6)}` ?? '']?.count as number ?? 0
```

Comment:
given the ternary on line 54, `feedPath` can potentially be `undefined` when calling `slice`